src/file/storage/storage.config.ts

```diff
@@ -0,0 +1,8 @@
+import { config } from '../../config';
+
+export const storageURL : string = 'http://40.113.65.101:9000/';
+export const storageIP  : string = '40.113.65.101';
+export const bucketName : string = `${config.storage}`;
+export const accessKey  : string = 'sealminio';
+export const secretKey  : string = '/QixUY2oU6YY0f+LSjbVsmlFxfUqSn5LA7x26nSVUDKJcU7kn9kxNRY+wdLNDHBWfxnOdR8SO1XmQ2A5NRr2Uw==';
```

Comment:
Open issue. Secret keys should not be on github.

Reply:
#69